src/FffGcodeWriter.cpp

```diff
@@ -2930,20 +2929,18 @@ bool FffGcodeWriter::processInsets(
     if (Application::getInstance().current_slice_->scene.current_mesh_group->settings.get<bool>("magic_spiralize"))
     {
         const size_t initial_bottom_layers = mesh.settings.get<size_t>("initial_bottom_layers");
```

Comment:
Maybe initialize `initial_bottom_layers` as being a `LayerIndex` ? That would avoid the 3 subsequent casts. But maybe it is used otherwise.

Reply:
I didn't want to do that since that would introduce a `LayerIndex` to `size_t` cast (https://github.com/Ultimaker/CuraEngine/pull/2139/commits/14630e645174c470026574922104f6d0e8e18c08#diff-1503d7ea10a5771cbd5934a56564ffae85b86613f443dd7e4750810dd8c58f5fR2949). But added it here https://github.com/Ultimaker/CuraEngine/pull/2139/commits/14630e645174c470026574922104f6d0e8e18c08